packages/nextjs-template/components/DendronSEO.tsx

```diff
@@ -68,7 +70,8 @@ export default function DendronSEO({
 
   const title = cleanSeoProps.title;
   const description = cleanSeoProps.excerpt;
-  const images = cleanSeoProps?.image ? [cleanSeoProps.image] : [];
+  const defaultImages = siteConfig?.image ? [siteConfig.image] : [];
+  const images = cleanSeoProps?.image ? [cleanSeoProps.image] : defaultImages;
```

Comment:
shouldn't line 69 already do this?
`_.defaults(noteSeoProps, siteSeoProps)`?

If not, you might want to use `_.defaultsDeep`

Reply:
It doesn't include `image` but I updated `getSEOPropsFromConfig` instead